TezosKit/Michelson/MichelsonParameter.swift

```diff
@@ -0,0 +1,28 @@
+// Copyright Keefer Taylor, 2019.
+
+import Foundation
+
+/// String constants used in Micheline param JSON encoding.
+internal enum MichelineConstants {
+  public static let args = "args"
```

Comment:
These consts might be easier to understand if they were grouped by kind. prim is a Micheline JSON key, as is int, but Pair is a keyword.

Reply:
I actually think that because sometimes type is a value (i.e. Pair) and sometimes it's a key (i.e. int) that this doesn't really break down well.